### ISSUE TITLE
Add the README.org file to the package archive installed by Emacs

### ISSUE THREAD

**Issue by rswgnu** [emacs-lsp/dap-mode#91]
When using the Emacs package manager to install, I did not see any documentation.

The documentation needs to be updated to explain how to invoke dap-debugger with command-line arguments and how to display the full ui shown here.  I did not see any doc on that.

**Comment by yyoncho** [comment]
If I understood it correctly you are asking for duplicating the README instructions in to package header? If that is the case, I would prefer not to do that since this will increase the amount of work that we have to do in order to keep the package up to date on two places. I checked a few popular packages (cider/magit) and they have taken this approach too.

**Comment by rswgnu** [comment]
> If I understood it correctly you are asking for duplicating the README instructions in to package header? 
> 
No, just have that file included in the Melpa distribution that is installed by the emacs package manager.  Many users might never visit the source archive on Github.
